packages/container/src/components/FieldPluginContainer.tsx

```diff
@@ -111,15 +112,17 @@ const useSandbox = (
   const [schema, setSchema] = useState<FieldPluginSchema>({
     field_type: 'preview',
     options: manifest.options,
+    translatable: false,
   })
   const [content, setContent] = useState<unknown>(initialContent)
-  const [language, setLanguage] = useState<string>('')
+  const [language, setLanguage] = useState<string>('default')
   const [stateChangedCallbackId, setStateChangedCallbackId] = useState<string>()
 
   const stateChangedData = useMemo<StateChangedMessage>(
     () => ({
       model: content,
       schema: schema,
+      interfaceLanguage: 'en',
```

Comment:
This was added due to type issues of `StateChangedMessage` which forced me to add this line of code.